cv32e40p/tests/programs/custom/pulp_hardware_loop/pulp_hardware_loop.S

```diff
@@ -0,0 +1,313 @@
+#
```

Comment:
I am assuming that you have modified this file.  If so, please add a copyright (with year) and an SPDX header.

Reply:
Created from scratch.

For all file in tests/programs/customs/* there is not really a common strategy about those headers.
But sure I can use the canvas I put on previous tests I created from scratch and added to core-v-verif.